### PR TITLE
Update User-Agent Header

### PR DESCRIPTION
**Pull request by smusali**
- remove unnecessary `get-distro`, `get-os`, and related test stuff
- update capturing `user-agent`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -228,38 +227,27 @@ if ((os.platform() === 'win32' && require('is-administrator')()) || process.getu
                         return utils.log(`error while saving to: ${conf_file}: ${error}`, 'error');
                     }
 
-                    saveMessages.forEach((message) => {
-                        console.log(message);
-                    });
-
+                    saveMessages.forEach((message) => { console.log(message); });
```
Comment (weizou19) [inline index.js]: or simply
```
saveMessages.forEach(console.log);
```

File: index.js [hunk 2]
```diff
@@ -299,10 +287,11 @@ if ((os.platform() === 'win32' && require('is-administrator')()) || process.getu
             }
         }
 
-        utils.log(`${program._name} ${pkg.version} started on ${config.hostname} (${config.ip})`);
-
-        if (config.platform && config.platform.indexOf('k8s') === 0) {
-            k8s.init();
+        utils.log(`${program._name}/${pkg.version} started on ${config.hostname} (${config.ip})`);
+        if (config.platform && config.platform.indexOf('k8s') === 0) { k8s.init(); }
```
Comment (weizou19) [inline index.js]: same as
```
if (config.platform && config.platform.startsWith('k8s') === 0)
```

File: index.js (hunk 1 above)
Comment (smusali) [reply index.js]:
```
> let saveMessages = ['message1', 'message2', 'message3'];
> saveMessages.forEach(console.log)
message1 0 [ 'message1', 'message2', 'message3' ]
message2 1 [ 'message1', 'message2', 'message3' ]
message3 2 [ 'message1', 'message2', 'message3' ]
```

File: index.js (hunk 2 above)
Comment (smusali) [reply index.js]: Done!

File: index.js (hunk 1 above)
Comment (weizou19) [reply index.js]: Ah, I see. Thanks for sharing

**Pull request merged**